server/controllers/inventory/inventory/core.js

```diff
@@ -143,6 +144,7 @@ function getItemsMetadata(params) {
   filters.equals('consumable');
   filters.equals('locked');
   filters.equals('label');
+  filters.equals('is_brocken');
```

Comment:
You should include `filters.fullText('note')` here, so that filtering on the `note` column works.